reana_workflow_controller/rest/workflows_status.py

```diff
@@ -522,7 +522,16 @@ def set_workflow_status(workflow_id_or_name):  # noqa
             )
         elif status == DELETED:
             all_runs = True if request.json.get("all_runs") else False
-            workspace = True if request.json.get("workspace") else False
+            workspace = True if request.json.get("workspace", True) else False
```

Comment:
I think we can leave this line as it was before, so that we don't change the default value of `workspace`. WDYT?

Reply:
In this case, when `workspace` is not specified, it will become `False` and fail.